### ISSUE TITLE
502 Bad Gateway after installing using script?

### ISSUE THREAD

**Issue by dma9** [causefx/Organizr#759]
###### Operating System: Ubuntu 17.10
<hr>

##### Problem Description:

With apologies in advance for being a bit of a noob: I'm trying to install Organizr in an LXC container for a headless server. I followed the instructions for a Linux installation using the script and all seemed to go well. I didn't notice any error messages. But when I try the last step (from a different computer using the IP address of the server), I get "502 Bad Gateway". I also tried installing Lynx locally on the server and tried localhost, 127.0.0.1 and the IP address of the server, but still get the same error message. Not quite sure what I'm doing wrong or if I missed a configuration step. Already reviewed all the subreddit as well as the issues and wiki here, but can't seem to figure out how to fix. Any suggestions would be most appreciated.
<hr>

##### Reproduction Steps:
As noted above.

<hr>

#### Errors on screen?  If so paste here:

```

```

**Comment by causefx** [comment]
Can you log in to the server and see Nginx logs?

**Comment by dma9** [comment]
Thanks for the quick response. There are a bunch of errors in /var/log/nginx/error.log, all of which follow the same pattern:

`2018/01/04 20:05:16 [crit] 12470#12470: *1 connect() to unix:/run/php/php7.0-fpm.sock failed (2: No such file or directory) while connecting to upstream, client: [IP address of router], server: www.[domain name], request: "GET /check.php HTTP/1.1", upstream: "fastcgi://unix:/run/php/php7.0-fpm.sock:", host: "[IP address of server]"
`

I noticed that the server was identified as "www.[domain name]" but the hostname of the server is actually "organizer.[domain name]" so I changed /etc/nginx/sites_available/[domain name].conf accordingly and restarted nginx (and then the entire container), thinking that might have something to do with it, but still get the same error.

The access log shows a bunch of get requests. I've excerpted a couple of lines:

`[IP address of router] - - [04/Jan/2018:20:05:16 -0500] "GET /check.php HTTP/1.1" 502 182 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:57.0) Gecko/20100101 Firefox/57.0"
`

`
[IP address of router] - - [04/Jan/2018:20:05:16 -0500] "GET /favicon.ico HTTP/1.1" 502 182 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:57.0) Gecko/20100101 Firefox/57.0"
`

Do let me know what you think.

**Comment by causefx** [comment]
When I get home I will take a look. I’m away until Saturday.

**Comment by dma9** [comment]
I'm terribly sorry for troubling you - I should have probably been a bit more diligent before posting. In looking at the log entries, I should have had the common sense to investigate why the connection to /run/php/php7.0-fpm.sock failed. In doing so, I discovered that it was /run/php/php7.**1**-fpm.sock there instead. So I changed /etc/nginx/config/phpblock.conf to point to php7.1-fpm.sock instead and restarted nginx and now it works. Thanks for pointing me in the right direction. I'm not quite sure how  or why it was pointing to the wrong .sock but hopefully this might be of some use if someone else runs into this issue.
